base/src/ipfixcol.c

```diff
@@ -255,27 +255,39 @@ int main (int argc, char* argv[])
 		MSG_SYSLOG_INIT(PACKAGE);
 		
 		/* and send all following messages to the syslog */
-		if (daemon (1, 0)) {
+		if (daemon(1, 0)) {
 			MSG_ERROR(msg_module, "%s", strerror(errno));
 		}
 	}
 
-	/* check config file */
-	if (config_file == NULL) {
+	/* Check config file */
+	if (startup_config == NULL) {
 		/* and use default if not specified */
-		config_file = DEFAULT_CONFIG_FILE;
-		MSG_NOTICE(msg_module, "Using default configuration file: %s", config_file);
+		startup_config = DEFAULT_STARTUP_CONFIG;
+		MSG_NOTICE(msg_module, "Using default configuration file: %s", startup_config);
+	} else if (access(startup_config, R_OK) == -1) {
```

Comment:
I fully agree with your proposal. For now, I have reverted my changes such that we can merge the remaining changes in this pull request.